config/locales/server.en.yml

```diff
@@ -188,7 +188,6 @@ en:
   topic_not_found: "Something has gone wrong. Perhaps this topic was closed or deleted while you were looking at it?"
 
   just_posted_that: "is too similar to what you recently posted"
-  has_already_been_used: "has already been used"
```

Comment:
Exactly, this is a duplicate. `errors.messages.has_already_been_used` is the path, not this one.

1. Validator invoke `errors.add` with `:taken` and customized message key: https://github.com/rails/rails/blob/v4.2.6/activemodel/lib/active_model/errors.rb#L299
2. It's fetched later: https://github.com/rails/rails/blob/v4.2.6/activemodel/lib/active_model/errors.rb#L413
3. key path is defined here.